### ISSUE TITLE
Bug: Unable to order CCK field weight (javascript issue)

### ISSUE THREAD

**Issue by rjcorwin** [jywarren/plots#50]
Here's what it looks like now.
<img src=http://rjsteinert.com/github/plot-cck-error2.png>
This is what it's missing.
<img src=http://rjsteinert.com/github/plot-cck-error1.png>

We could turn the default Garland theme on for the Admin section as I have done in my sandbox http://publiclaboratory.org/admin/settings/admin.

**Comment by jywarren** [comment]
This one is a doozy. jquery sparklines (and other modern jquery features) depend on a pretty recent version of jquery. This same recent version (which i added in plots.info) makes the drag & drop stop working. I guess we could do a conditional include based on whether "/admin/" is in the path? For the past few days, i've been commenting out the newer version whenever i want to reorder things. Dumb.

**Comment by rjcorwin** [comment]
jQuery Update module (http://drupal.org/project/jquery_update) attempts to handle these backward compatibility issues.  I'll give a try in my sandbox.  If managing versions of jQuery becomes hell, then something like the GitHub graphs could be in an iframe passing in parameters to an html file with it's own js includes with javascript that pulls JSON from a View using the Views Datasource module.

**Comment by jywarren** [comment]
or we could even assign "new jquery" a different $ selector.

On Wed, Dec 21, 2011 at 11:37 AM, R.J. Steinert (rjstatic) <
reply@reply.github.com

**Comment by rjcorwin** [comment]
True dat.  I just tried uncommenting your jquery include in plots.info and enabling the jquery_update module.  Draggables, sparklines, and drop down menus are all working.  It looks like it's using jQuery 1.3.2 which is a step down from 1.7.1, but hey, it works.  Is there anything else I should test? It's hard to tell on map, I still have the API key error.

**Comment by jywarren** [comment]
is this on your sandbox?

the only other places i can think of are the Table of Contents feature and
the front page image carousel.

http://publiclaboratory.org/licenses has a TOC, its working on the live site

On Wed, Dec 21, 2011 at 12:31 PM, R.J. Steinert (rjstatic) <
reply@reply.github.com

**Comment by rjcorwin** [comment]
Haha, ya, in my sandbox.  I'll check those other places.  What is a good example of for the TOC?  I have a fresh db and code base as of this morning.

**Comment by rjcorwin** [comment]
Oh sorry, there's the TOC example in your comment. Thanks.

**Comment by jywarren** [comment]
seeing more trouble due to jquery version fragmentation... https://github.com/jywarren/plots/issues/69 was traced to this. Are we running jquery_update on the live site?

**Comment by rjcorwin** [comment]
Nope, we don't have the jquery_update module in this repo yet.  Only your updates in the plots.info file https://github.com/jywarren/plots/blame/master/html/sites/all/themes/plots/plots.info

**Comment by jywarren** [comment]
OK, do you want to try using jquery_update? I'm wgetting it into the
modules folder now (the "good" modules folder, as you've set a good example
:-P)

On Sun, Feb 12, 2012 at 10:00 PM, R.J. Steinert <
reply@reply.github.com

**Comment by rjcorwin** [comment]
jquery update will surely break things but we'll have to deal with the
technical debt of having an outdated version of jquery eventually.  Is now
a good time (is it ever :P)?

On Mon, Feb 13, 2012 at 5:45 PM, Jeffrey Warren <
reply@reply.github.com

**Comment by jywarren** [comment]
OK, I've listed the jquery dependent code I know of; we ought to try to upgrade to a fairly recent version and make sure these work:

http://publiclaboratory.org/wiki/jquery-dependencies

We maybe should also list which of the above work on 1.7.x vs. the native jquery from drupal...
